docs/platform/howto/make-super-admin.md

```diff
@@ -10,6 +10,9 @@ The super admin role is a special role that has unrestricted access to an organi
 This role should be limited to as few users as possible for organization setup and emergency use. For daily administrative tasks, assign users the [organization admin role](/docs/platform/concepts/permissions) instead. Aiven also highly recommends enabling [two-factor authentication](/docs/platform/howto/user-2fa) for super admin.
 :::
 
+Only super admin can grant or revoke super admin privileges for other users. Super admin
+are also the only users that can delete an organization.
```

Comment:
Either

```suggestion
Only super admin can grant or revoke super admin privileges for other users. The super admin
is also the only user that can delete an organization.
```
or

```suggestion
Only super admin can grant or revoke super admin privileges for other users. Super admins
are also the only users that can delete organizations.
```

Reply:
There isn't only one super admin so I wouldn't go with the first suggestion. As for the second suggestion, the terminology that we've standardized on for administrator roles is uncountable `admin`. That's true for organization admin, project admin, and super admin. The plural is `admin`.

Reply:
Ok, looked pretty off to me but was not aware of this agreement. Thanks for the clarification 🙂